doc/development/developing-sql-dialect/implementing_mandatory_sql_dialect_classes.md

```diff
@@ -339,7 +339,7 @@ And we also need two corresponding test classes:
 
     @Override
     protected QueryRewriter createQueryRewriter() {
-        return new BaseQueryRewriter(this, this.remoteMetadataReader, this.connectionFactory);
```

Comment:
Propose this change in the main repository, since it is independent of a dialect.
Remove the file in this repository afterwards.